e2e/pages/adf/process-cloud/processListCloudComponent.ts

```diff
@@ -23,6 +23,21 @@ export class ProcessListCloudComponent {
 
     processList = element(by.css('adf-cloud-process-list'));
     noProcessFound = element.all(by.css("p[class='adf-empty-content__title']")).first();
+    noOfColumns = element.all(by.css('adf-cloud-process-list adf-datatable .adf-datatable-header .adf-datatable-row .adf-datatable-table-cell-header'));
```

Comment:
This should be allColumns, not numberOfColumns. Could you also change the locator ? You can use the data-automation-id

Reply:
Please don't use datatable locators inside other page object. Create a method inside datatable PO and call it here, in case doesn't exist already.

Reply:
done, changed the name and moved it to DataTablePage.

Reply:
changed the locator to use data-automation-id